docs/user_guide/assessment.rst

```diff
@@ -470,6 +470,19 @@ There are many other customizations that can be done. More information can be fo
 
 .. _dashboard:
 
+
+Plotting model comparisons
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+To compare many different models using certain metrics, Fairlearn provides
+an easy to use interface.
+
+.. literalinclude:: ../auto_examples/plot_grid_search_census.py
+    :language: python
+    :start-after: # Plot model comparison
```

Comment:
Nit: perhaps we should add an end-before, too, just in case that example is extended in the future.

Reply:
That seems smart! I also added it to the one above because